test/server/cards/15.3-ACO/DamnedHida.spec.js

```diff
@@ -0,0 +1,30 @@
+describe('Damned Hida', function() {
+    integration(function() {
+        beforeEach(function() {
+            this.setupTest({
+                phase: 'conflict',
+                player1: {
+                    inPlay: ['damned-hida']
+                },
+                player2:{
+                    inPlay: ['damned-hida']
+                }
+            });
+
+            this.hida = this.player1.findCardByName('damned-hida');
+            this.hida2 = this.player2.findCardByName('damned-hida');
+            this.hida2.fate = 1;
+
+        });
+
+
+        it('Gets a +3 military skill when dire', function() {
+            expect(this.hida.getMilitarySkill()).toBe(this.hida.printedMilitarySkill + 3);
+        });
+
+        it('should not get +3 military skill when dire', function() {
+            expect(this.hida2.getMilitarySkill()).toBe(this.hida2.printedMilitarySkill + 3);
+        });
```

Comment:
The 2nd test is wrong (it's getting the dire effect when you're trying to test it without the dire effect).  You probably need a `checkGameState` after setting the fate to 1